### PR TITLE
docs: Documentation for backup feature

### PR DESCRIPTION
**Pull request by hackerchai**

### CODE REVIEW

File: docs/pages/deploy/backup.zh-CN.mdx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ogImageTitle: 备份服务
+ogImageSubtitle: 备份你的服务
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+# 备份服务
+
+在生产环境和开发过程中，您可能需要为您的  创建备份。这会对应用的健壮性和可靠性产生积极的影响。
+我们为所有部署在 Zeabur 上的 [Prebuilt Service](/prebuilt/intro) 提供了一个简单的方法来创建和恢复备份。
```
Comment (pan93412) [inline docs/pages/deploy/backup.zh-CN.mdx:12]:
```suggestion
在生产环境和开发过程中，您可能需要为您的服务创建备份，增强应用的健壮性和可靠性。
我们为所有部署在 Zeabur 上的 [Prebuilt Service](/prebuilt/intro) 提供了一个简单的方法来创建和恢复备份。
```

**Review by pan93412 — CHANGES_REQUESTED** [review]

File: docs/pages/deploy/backup.zh-CN.mdx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+<Callout>
+    上方提到的 [Prebuilt Service](/prebuilt/intro) , 是属于服务下的一个子分类，[Prebuilt Service](/prebuilt/intro) 是有状态服务，不同于 `Git Service` 是无状态的， 所以 `Git Service` 不支持备份功能
+</Callout>
+
+## 步骤一：挂起你的服务
```
Comment (MichaelYuhe) [inline docs/pages/deploy/backup.zh-CN.mdx:18]: 挂起 -> 暂停合适一些

**Review by MichaelYuhe — CHANGES_REQUESTED** [review]

**Review by MichaelYuhe — APPROVED** [review]

**Comment by MichaelYuhe** [comment]
cc @pan93412 for review

File: docs/pages/deploy/backup.zh-TW.mdx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+import { Callout } from 'nextra-theme-docs'
+
+# 備份服務
+
+在生產環境和開發過程中，您可能需要為您的服務建立備份，增強應用的健壯性和可靠性。
+我們為所有部署在 Zeabur 上的 [Prebuilt Service](/prebuilt/intro) 提供了一個簡單的方法來建立和復原備份。
+
+<Callout>
+    上方提到的 [Prebuilt Service](/prebuilt/intro) , 是屬於服務下的一個子分類，[Prebuilt Service](/prebuilt/intro) 是有狀態服務，不同於 `Git Service` 是無狀態的， 所以 `Git Service` 不支援備份功能
```
Comment (pan93412) [inline docs/pages/deploy/backup.zh-TW.mdx:15]:
```suggestion
    上方提到的 [Prebuilt Service](/prebuilt/intro) , 是屬於服務下的一個子分類，[Prebuilt Service](/prebuilt/intro) 是有狀態服務，不同於 `Git Service` 是無狀態的， 所以 `Git Service` 不支援備份功能。
```

File: docs/pages/deploy/backup.zh-CN.mdx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+import { Callout } from 'nextra-theme-docs'
+
+# 备份服务
+
+在生产环境和开发过程中，您可能需要为您的服务创建备份，增强应用的健壮性和可靠性。
+我们为所有部署在 Zeabur 上的 [Prebuilt Service](/prebuilt/intro) 提供了一个简单的方法来创建和恢复备份。
+
+<Callout>
+    上方提到的 [Prebuilt Service](/prebuilt/intro) , 是属于服务下的一个子分类，[Prebuilt Service](/prebuilt/intro) 是有状态服务，不同于 `Git Service` 是无状态的， 所以 `Git Service` 不支持备份功能
```
Comment (pan93412) [inline docs/pages/deploy/backup.zh-CN.mdx:15]:
```suggestion
    上方提到的 [Prebuilt Service](/prebuilt/intro) , 是属于服务下的一个子分类，[Prebuilt Service](/prebuilt/intro) 是有状态服务，不同于 `Git Service` 是无状态的， 所以 `Git Service` 不支持备份功能。
```

File: docs/pages/deploy/backup.zh-TW.mdx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+</Callout>
+
+## 步驟一：暫停你的服務
+
+在備份之前請先暫停你的服務，以免在備份過程遺失資料，如果你的服務是執行狀態，系統會提示你暫停服務。
+
+![backup-not-suspend](/deploy/backup/backup-not-suspend.png)
+
+此時你可以打開 **Settings** 頁籤，找到 `Danger Zone` 區域， 點擊 `Suspend Service` 按鈕暫停你的服務。
```
Comment (pan93412) [inline docs/pages/deploy/backup.zh-TW.mdx:24]:
```suggestion
此時你可以打開 **Settings** 區塊，找到 `Danger Zone` 區域， 點選 `Suspend Service` 按鈕暫停你的服務。
```

File: docs/pages/deploy/backup.zh-TW.mdx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+![suspend](/deploy/backup/backup-suspend-service.png)
+
+## 步驟二：打開備份設定
+
+選擇你想要進行備份的服務，展開 **Backup** 頁籤，你會看到一個按鈕，用於為你的服務建立備份。
```
Comment (pan93412) [inline docs/pages/deploy/backup.zh-TW.mdx:30]:
```suggestion
選擇你想要進行備份的服務，展開 **Backup** 區塊，你會看到一個按鈕，用於為你的服務建立備份。
```

File: docs/pages/deploy/backup.zh-TW.mdx [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+## 步驟三：建立備份
+
+點擊 **Backup** 按鈕，系統會為你的服務建立一個備份，這個過程可能需要一些時間。
+
+![created](/deploy/backup/backup-created.png)
+
+## 步驟四：下載備份
+
+等待一段時間後，你可以看到備份已經建立完成，**Backup** 頁籤上會顯示 `SUCCESS` ，點擊此處即可下載備份文件。
```
Comment (pan93412) [inline docs/pages/deploy/backup.zh-TW.mdx:42]:
```suggestion
等待一段時間後，你可以看到備份已經建立完成，**Backup** 區塊上會顯示 `SUCCESS` ，點選此處即可下載備份檔案。
```

File: docs/pages/deploy/backup.zh-TW.mdx [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+![success](/deploy/backup/backup-success.png)
+
+得到下載的文件後，你可以將其解壓縮並查看其中的內容。
```
Comment (pan93412) [inline docs/pages/deploy/backup.zh-TW.mdx:46]:
```suggestion
得到下載的檔案後，你可以將其解壓縮並檢閱其中的內容。
```

**Review by pan93412 — CHANGES_REQUESTED** [review]

**Review by MichaelYuhe — APPROVED** [review]

**Review by pan93412 — APPROVED** [review]

**Pull request merged**